# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/clanlib/clanlib06/clanlib_glxfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Sources/GL/GLX/displaycard_glx.cpp	2003-04-28 10:07:51.000000000 +0200
+++ ./Sources/GL/GLX/displaycard_glx.cpp	2003-04-28 10:11:54.000000000 +0200
@@ -18,6 +18,9 @@
 #include <X11/Xutil.h>
 #include <X11/cursorfont.h>
 
+#define GLX_GLXEXT_PROTOTYPES
+#include <GL/glx.h>
+
 #include "displaycard_glx.h"
 #include "GL/Generic/blit_gl_generic.h"
 #include "API/Core/System/cl_assert.h"
